astroid/nodes/node_classes.py

```diff
@@ -4728,6 +4756,30 @@ def postinit(self, values: list[NodeNG] | None = None) -> None:
     def get_children(self):
         yield from self.values
 
+    def _infer(
+        self, context: InferenceContext | None = None, **kwargs: Any
+    ) -> Generator[InferenceResult, None, InferenceErrorInfo | None]:
+        yield from self._infer_from_values(self.values)
+
+    @classmethod
+    def _infer_from_values(
+        cls, nodes: list[NodeNG], context: InferenceContext | None = None, **kwargs: Any
+    ) -> Generator[InferenceResult, None, InferenceErrorInfo | None]:
+        if len(nodes) == 1:
+            yield from nodes[0]._infer(context, **kwargs)
+            return
+        for prefix in nodes[0]._infer(context, **kwargs):
+            for suffix in cls._infer_from_values(nodes[1:], context, **kwargs):
+                result = ""
+                for node in (prefix, suffix):
+                    if node is util.Uninferable:
+                        result += "{Uninferable}"
+                    elif isinstance(node, Const):
+                        result += str(node.value)
+                    else:
+                        result += node.as_string()
```

Comment:
Also missing coverage.

Reply:
This can never happen, line is now removed